docs/RFCS/online_schema_change.md

```diff
@@ -131,8 +131,24 @@ Since schema change operations are potentially long running they need
 to be restartable or abortable if the node performing them dies. We
 accomplish this by performing the schema change operation for a table
 on a well known node: the replica holding the leader lease for the
```

Comment:
When a table is not accessed, it's possible that no leader lease is obtained and so the migration would be dormant. This is extremely unlikely (and if so, it'd be picked back up when someone tried to do anything with the table). If that's ever a concern, we could just have a goroutine on the descriptor table which reads the first table key for migrations which haven't been pinged for a long time.